File: java praktikum/prak11/JavaApplication19/src/javaapplication19/Soal2.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package javaapplication19;

import java.util.Scanner;

/**
 *
 * @author Nrhys
 */
public class Soal2 {
    public static void main(String[] args) {
        double bb; 
        int k = 0
                , l= 0
                , s = 0
                , g = 0
                , o = 0;
        Scanner input = new Scanner(System.in);
        
        for(int i = 0; i < 10; i++){
            System.out.println("Inputkan Berat Bebek ke-"+(i+1));
            bb = input.nextDouble();
            
            if(bb > 2){
                System.out.println("Bebek ke-"+(i+1)+" Obesitas");
                o += 1;
            }else if(bb >= 1.5){
                System.out.println("Bebek ke-"+(i+1)+" Gemuk");
                g += 1;
            }else if(bb >= 1){
                System.out.println("Bebek ke-"+(i+1)+" Standard");
                s += 1;
            }else if(bb >= 0.2){
                System.out.println("Bebek ke-"+(i+1)+" Langsing");
                l += 1;
            }else {
                System.out.println("Bebek ke-"+(i+1)+" Kurus");
                k += 1;
            }
            
            
        }
        
        System.out.println("Bebek Kurus ada " +k + " ekor");
        System.out.println("Bebek Langsing ada "+l + " ekor");
        System.out.println("Bebek Standard ada "+s + " ekor");
        System.out.println("Bebek Gemuk ada "+g + " ekor");
        System.out.println("Bebek Obesistas ada "+o + " ekor");
    }
}

File: Python praktikum/prak4/Soal2.py
k = 0
l = 0
s = 0
g = 0
o = 0

for i  in range(10):
    bb = float(input("Masukan Berat Bebek ke-" + str(i+1)+ " "))

    if bb > 2 :
        print("Bebek ke-" + str(i+1)+" Obesitas")
        o += 1
    elif bb >= 1.5:
        print("Bebek ke-" + str(i+1)+" Gemuk")
        g += 1
    elif bb >= 1:
        print("Bebek ke-" + str(i+1)+" Standard")
        s += 1
    elif bb >= 0.2:
        print("Bebek ke-" + str(i+1)+" Langsing")
        l += 1
    else:
        print("Bebek ke-" + str(i+1)+" Kurus")
        k += 1


print("Bebek Kurus ada " + str(k) + " ekor")
print("Bebek Langsing ada " + str(l) + " ekor")
print("Bebek Standard ada " + str(s) + " ekor")
print("Bebek Gemuk ada " + str(g) + " ekor")
print("Bebek Obesitas ada " + str(o) + " ekor")
